docs/xmlprofiles.rst

```diff
@@ -0,0 +1,1048 @@
++-----------------------------------+-------------------------------------------------------------------------+-------------------------------------+-----------------------+
+| ``<throughputController>``        | Limits the output bandwidth of the publisher.                           | `Throughput`_                       |                       |
++-----------------------------------+-------------------------------------------------------------------------+-------------------------------------+-----------------------+
+| ``<historyMemoryPolicy>``         | Memory allocation kind for pubsliher's history.                         | :class:`PREALLOCATED`,              | :class:`PREALLOCATED` |
+|                                   |                                                                         | :class:`PREALLOCATED_WITH_REALLOC`, |                       |
+|                                   |                                                                         | :class:`DYNAMIC`                    |                       |
++-----------------------------------+-------------------------------------------------------------------------+-------------------------------------+-----------------------+
+| ``<propertiesPolicy>``            | Additional configuration properties.                                    | :ref:`PropertiesPolicyType`         |                       |
++-----------------------------------+-------------------------------------------------------------------------+-------------------------------------+-----------------------+
+| ``<userDefinedID>``               | Used for StaticEndpointDiscovery.                                       | ``Int16``                           | -1                    |
++-----------------------------------+-------------------------------------------------------------------------+-------------------------------------+-----------------------+
+| ``<entityID>``                    | EntityId of the *endpoint*.                                             | ``Int16``                           | -1                    |
++-----------------------------------+-------------------------------------------------------------------------+-------------------------------------+-----------------------+
+
+.. _pubtimes:
+
+**Times**
+
++------------------------------+-------------------------------------------------------------------------------+----------------------------------+--------------------+
+| Name                         | Description                                                                   | Values                           | Default            |
++==============================+===============================================================================+==================================+====================+
+| ``<initialAcknackDelay>``    | Initial heartbeat delay.                                                      | :ref:`DurationType`              | ~45 ms             |
```

Comment:
```suggestion
| ``<initialHeartbeatDelay>``    | Initial heartbeat delay.                                                      | :ref:`DurationType`              | ~45 ms             |
```

Reply:
Changed, but not with your suggestion as it doesn't respect the table's spaces.